contrib/win-installer/podman.wxs

```diff
@@ -29,6 +29,9 @@
             </Component>
             <Component Id="WinSshProxyExecutable" Guid="0DA730AB-2F97-40E8-A8FC-356E88EAA4D2" Win64="yes">
               <File Id="WinSshProxyExecutableFile" Name="win-sshproxy.exe" Source="artifacts/win-sshproxy.exe" KeyPath="yes"/>
+            </Component>
+             <Component Id="GvProxyExecutable" Guid="1A4A2975-AD2D-44AA-974B-9B343C098333" Win64="yes">
```

Comment:
@arixmkii just had a thought here, what if we just surrounded these gvproxy sections with a pre-processor conditional directive e.g. `<?if $(env.UseGVProxy) != Skip?>`

The powershell release process script can then just set it as a return value passed as a param to the build. If you don't have the bandwidth I could throw that on. Let me know.

Reply:
Added in a separate commit.

Implementation:
* In build script: check if there is no gvproxy, then don't try to sign it and set env var to Skip for child processes
* in Wix xml: conditionally check for presence of env var and if it exists then use it as a value for the variable, if not then just fallback to default value - empty string. This is needed to workaround undefined variable expansion, because to me it looked like missing env var should not fail the build.

Reply:
I didn't apply similar fix to old `podman.msi`, because building from the actual source tree it will have gvproxy.exe starting from the same commit.

Reply:
Excellent. This change looks good to me. Great work. Just to confirm you manually tested both scenarios? 

I agree with the old msi not being necessary for this change, since its only built as a one-off. The plan is to remove it soon anyway.  It was just left behind until we knew the new installer was in wide use.

Reply:
Yes. I used zip archive built locally and the artifact from v4.3.1 stable to test both scenarios. And then installed it on my test system to verify extracted content in Program Files folder.
It actually took me a number of iterations to get both of them working 😅  So, here I'm really happy that I did these tests.

Reply:
cool thanks. yeah it's amazing how finicky the installer technology stack is for Windows!